ARIES_AUTORENEWAL_RFP.md

```diff
@@ -868,8 +866,8 @@ in the last five (5) years in a technical leadership role.
 
 ## Sec. 3.04 Contract Term and Work Schedule
 
-The term of this contract will be from approximately December 15, 2021
--- September 30, 2022. Any potential for contract amendments that alter
+The term of this contract will be from approximately January 3, 2021
```

Comment:
Should be = to Contract Start? January 10, 2022

Reply:
fixed